src/app/frontend/common/components/responsivetable/responsivetable_directive.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Returns directive definition for responsive table.
+ *
+ * @return {!angular.Directive}
+ */
+export default function responsiveTableDirective() {
+  return {
+    restrict: 'A',
+    compile: function(element, attrs) {
```

Comment:
I see and I think it's pretty nice way to avoid adding labels directly for each table like author of https://css-tricks.com/responsive-data-tables/ does:

```css
td:nth-of-type(1):before { content: "First Name"; }
td:nth-of-type(2):before { content: "Last Name"; }
td:nth-of-type(3):before { content: "Job Title"; }
...
td:nth-of-type(10):before { content: "Arbitrary Data"; }
```

Instead of this we got just `content: attr(data-title);` in stylesheet. Do you see any other generic solution for it?

Regarding code, it's on MIT license, but anyways, I forgot to put an information about author in source code. Sorry!